index.html

```diff
@@ -154,43 +186,73 @@ <h3 class="bottom-connect">LET’S CONNECT</h3>
                         <li class="name-one">
                             <div class="language-arrow">
                                 <h3>Languages</h3>
-                                <img src="./Images/arrow.png" alt="arrow-image">
+                                <img src="./Images/arrow.png" alt="arrow-image" />
                             </div>
                             <ul class="skill-list">
                                 <li class="skills">
-                                    <img src="./Images/js-icon.png" alt="javascript-icon">
+                                    <img src="./Images/js-icon.png" alt="javascript-icon" />
                                     <p class="skill-para">JavaSript</p>
                                 </li>
                                 <li class="skills">
-                                    <img src="./Images/html.png" alt="html-icon">
+                                    <img src="./Images/html.png" alt="html-icon" />
                                     <p class="skill-para">HTML</p>
                                 </li>
                                 <li class="skills">
-                                    <img src="./Images/css3-icon.png" alt="css-icon">
+                                    <img src="./Images/css3-icon.png" alt="css-icon" />
                                     <p class="skill-para">CSS</p>
                                 </li>
                             </ul>
                         </li>
                         <li class="name-two">
                             <div class="framework-arrow">
                                 <h3>Frameworks</h3>
-                                <img src="./Images/arrow2.png" alt="arrow-image">
+                                <img src="./Images/arrow2.png" alt="arrow-image" />
                             </div>
-                            <hr>
+                            <hr />
                         </li>
                         <li class="name-three">
                             <div class="skill-arrow">
                                 <h3>Skills</h3>
-                                <img src="./Images/arrow2.png" alt="arrow-image">
+                                <img src="./Images/arrow2.png" alt="arrow-image" />
                             </div>
-                            <hr>
+                            <hr />
                         </li>
                     </ul>
-
                 </section>
             </section>
-        </section>
+            <footer class="card-contact">
```

Comment:
- [ ] [OPTIONAL] Even though this is the last section in your web app, it's not really a footer, so I would suggest applying a better tag to name this section, such as a `section` tag with a `class="contact"` or maybe a `div` tag using the same class. I understand your point about making it a footer, but just remember this for future projects, because a footer should be used to contain some information about the page that should not be placed on any other site, such as copyright, contact information, or similar.

Reply:
ok, will follow the suggestion.